stock_outgoing_shipment_report/models/stock_picking.py

```diff
@@ -18,12 +18,14 @@ def generate_stock_outgoing_shipment_report(self):
             product = move.product_id
             vals = {"move_id": move.id}
             if order:
-                carrier = order.carrier_id
+                carrier = self.carrier_id
                 vals["carrier_id"] = carrier.id if carrier else False
                 vals["ship_service_id"] = (
                     order.delivery_carrier_service_id
                     and order.delivery_carrier_service_id.id
                 )
+                vals["ship_carrier"] = carrier.name
+                vals["ship_billing"] = carrier.ship_billing
```

Comment:
```suggestion
```
These are not needed as relations in the report model take care of them.

Reply:
Delete it. Thank you.